components/st-microelectronics/landpatterns.stanza

```diff
@@ -181,3 +181,169 @@ public pcb-landpattern WLCSP168:
+                      tol(9.0, 0.2),   ;Outer size (lead-to-lead):  D and E
+                      tol(7.0, 0.2),   ;Package size:               D1 and E1
+                      tol(0.6, 0.15),  ;Lead size in x direction:   L
+                      tol(0.22, 0.05)) ;Lead size in y direction:   b
+
+public pcb-landpattern LQFP64:
+  make-qfp-landpattern(64,             ;Number of pins
+                      0.5,             ;Pitch:                      e
+                      tol(12.0, 0.0),  ;Outer size (lead-to-lead):  D and E
+                      tol(10.0, 0.0),  ;Package size:               D1 and E1
+                      tol(0.6, 0.15),  ;Lead size in x direction:   L
+                      tol(0.22, 0.05)) ;Lead size in y direction:   b
+
+public pcb-landpattern LQFP80:
+  make-qfp-landpattern(80,             ;Number of pins
+                      0.5,             ;Pitch:                      e
+                      tol(14.0, 0.0),  ;Outer size (lead-to-lead):  D and E
+                      tol(12.0, 0.0),  ;Package size:               D1 and E1
+                      tol(0.6, 0.15),  ;Lead size in x direction:   L
+                      tol(0.22, 0.05)) ;Lead size in y direction:   b
+
+;public pcb-landpattern UFQFPN20:
+;  make-qfn-landpattern(20,             ;Number of pins
+;                      0.5,             ;Pitch:                      e
+;                      tol(3.0, 0.1),   ;Outer size (lead-to-lead):  D and E
+;                      tol(0.55, 0.05), ;Lead size in x direction:   L1?
+;                      tol(0.25, 0.05)) ;Lead size in y direction:   b
+
+public pcb-landpattern UFQFPN28:
+  make-qfn-landpattern(28,             ;Number of pins
+                      0.5,             ;Pitch:                      e
+                      tol(4.0, 0.1),   ;Outer size (lead-to-lead):  D and E
+                      tol(0.4, 0.1),   ;Lead size in x direction:   L
+                      tol(0.25, 0.05)) ;Lead size in y direction:   b
+
+public pcb-landpattern UFQFPN32:
+  make-qfn-landpattern(32,             ;Number of pins
+                      0.5,             ;Pitch:                      e
+                      tol(5.0, 0.1),   ;Outer size (lead-to-lead):  D and E
+                      tol(0.4, 0.1),   ;Lead size in x direction:   L
+                      tol(0.23, 0.05), ;Lead size in y direction:   b
+                      Rectangle(3.45, 3.45)) ;GeneralChamferedRectangle(3.45, 3.45, [0.5 0. 0. 0.]))
+
+public pcb-landpattern UFQFPN48:
+  make-qfn-landpattern(48,             ;Number of pins
+                      0.5,             ;Pitch:                      e
+                      tol(7.0, 0.1),   ;Outer size (lead-to-lead):  D and E
+                      tol(0.4, 0.1),   ;Lead size in x direction:   L
+                      tol(0.25, 0.05), ;Lead size in y direction:   b
+                      GeneralChamferedRectangle(5.6, 5.6, [0.5 0. 0. 0.]))
+
+; public pcb-landpattern VFQFPN36:
```

Comment:
Remove comments

Reply:
Removed and tracking unsupported landpatterns locally